executor/adapter.go

```diff
@@ -267,15 +268,15 @@ func (a *ExecStmt) buildExecutor(ctx context.Context) (Executor, error) {
 func (a *ExecStmt) logSlowQuery() {
 	cfg := config.GetGlobalConfig()
 	costTime := time.Since(a.startTime)
-	sql := a.Text
+	sql := []rune(a.Text)
 	if len(sql) > cfg.Log.QueryLogMaxLen {
-		sql = sql[:cfg.Log.QueryLogMaxLen] + fmt.Sprintf("(len:%d)", len(sql))
+		sql = append(sql[:cfg.Log.QueryLogMaxLen], []rune(fmt.Sprintf("(len:%d)", len(sql)))...)
```

Comment:
How about 
```
fmt.Sprintf("%q(len:%d)", sql[:cfg.Log.QueryLogMaxLen], len(sql)
```

Reply:
%q will print the []rune like this:

```
['来' '来' '来']
```

Reply:
I changed `sql` type to string.

Reply:
sql := strconv.Quote(a.Text))

Reply:
So the fix is about the truncate make the last character illegal utf8?

Reply:
Yes, one of the problem.